ash/src/extensions/ext/acquire_drm_display.rs

```diff
@@ -0,0 +1,61 @@
+use crate::prelude::*;
+use crate::vk;
+use crate::{Entry, Instance};
+use std::ffi::CStr;
+use std::mem;
+
+/// <https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VK_EXT_acquire_drm_display.html>
+#[derive(Clone)]
+pub struct AcquireDrmDisplay {
+    handle: vk::Instance,
```

Comment:
No need to store `handle` if it isn't used by any of the functions wrapped by this struct. In turn drop `fn instance()` as well.

Reply:
removed in d855ed1